airflow/cli/commands/remote_commands/config_command.py

```diff
@@ -64,3 +67,390 @@ def get_value(args):
         print(value)
     except AirflowConfigException:
         pass
+
+
+class RenamedTo(NamedTuple):
```

Comment:
Probably could just name it as `ConfigParameter`? It does not store data related to the idea of rename

Reply:
Changed it